### DISCUSSION TITLE
gRPC server support through an extension

### DISCUSSION THREAD

**Discussion by akshatdubeysf** [loopbackio/loopback-next#7683] [Ideas]
I wanted to build a gRPC server using Loopback4, I tried to find an extension similar to the socketIo extension from [this](https://loopback.io/doc/en/lb4/apidocs.index.html) list, but could not find one.

If no such extension exists, I would be happy to contribute with some help and guidance.

If it does, it would be really helpful if someone could point me to it.

**Comment by achrinza** [comment]
There's an experimental GRPC extension here:

https://github.com/loopbackio/loopback4-extension-grpc

We haven't had the chance to transfer the package over to the monorepo yet, hence why it doesn't appear in the API docs.

Hope this helps!

**Reply by akshatdubeysf** [reply]
@achrinza Thanks!

**Reply by akshatdubeysf** [reply]
@achrinza one minor query, the types generated by this extension are adding author - Jonathan Casarrubias automatically, is their any config or something to change this behavior, or atleast the author name.
